### PR TITLE
DVHZ sensor swap

### PR DESCRIPTION
**Pull request by mnaguit**
like-for-like sensor swap
- from Lennartz MKIII (F-0517) to Lennartz MKIII (G-0572)
- for LC 10 (EH-ENZ) streams

### CODE REVIEW

**Review by CBurton90 — APPROVED** [review]

**Review by salichon — APPROVED** [review]

**Comment by mnaguit** [comment]
This is ready to be merged. But I am currently waiting for review of pending PRs (MHEZ and MWOZ) before I merge this, to avoid conflicts. FYI @CBurton90 @salichon

**Pull request merged**
